modules/oe_starter_content_news/config/install/core.entity_form_display.node.oe_news.default.yml

```diff
@@ -0,0 +1,97 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.field.node.oe_news.body
+    - field.field.node.oe_news.field_oe_featured_media
+    - field.field.node.oe_news.field_oe_publication_date
+    - field.field.node.oe_news.field_oe_summary
+    - node.type.oe_news
+  module:
+    - datetime
+    - text
+id: node.oe_news.default
+targetEntityType: node
+bundle: oe_news
+mode: default
+content:
+  body:
+    type: text_textarea_with_summary
+    weight: 123
```

Comment:
It's a normal weight? Maybe 10 or if is needed higher 50... same for the others. Starting in 120 not seems normal to me (maybe there is a reason for that)